src/Goerli-DssSpell.t.sol

```diff
@@ -23,8 +24,45 @@ interface DenyProxyLike {
     function denyProxy(address) external;
 }
 
+interface StarknetLike {
+    function ceiling() external returns (uint256);
+}
+
 contract DssSpellTest is GoerliDssSpellTestBase {
 
+    function testStarknetUpdates() public {
+
+        // Test before spell
+        // Currently 1M on Goerli, 100k on mainnet
+        assertEq(
+            StarknetLike(addr.addr("STARKNET_DAI_BRIDGE")).ceiling(),
```

Comment:
I'm getting a test failure here:

```haskell
Error: a == b not satisfied [uint]
    Expected: 1000000000000000000000000
      Actual: 50000000000000000000
```
at block 7016000

Reply:
Yes, someone on the starknet team recently updated this. We will need to examine the mainnet bridge to see if Starknet team is retaining any authority over that contract.

https://goerli.etherscan.io/tx/0xdc31797b14d7be07158bc95415e4d5a9e95d00725a9eda2d781db4949145e61d